app/services/request_password_reset.rb

```diff
@@ -16,15 +17,25 @@ def perform
   private
 
   def send_reset_password_instructions
+    irs_params = {
+      email: email,
+    }
+
     if Throttle.new(user: user, throttle_type: :reset_password_email).throttled_else_increment?
       analytics.throttler_rate_limit_triggered(throttle_type: :reset_password_email)
+      irs_params[:success] = true
+      irs_params[:failure_reason] = 'throttle rate limit triggered'
```

Comment:
should we come up with a "slug" name for this?

Reply:
Also, in other places, `failure_reason` is a hash, and that's what the YARD documentation says for this one, but this is a plain string, can we make the typed documentation and this value agree?

Reply:
Yes, I think I'll change the value to a hash. Unsure about the format at the moment.